src/data/set/basic.lean

```diff
@@ -1095,7 +1163,39 @@ theorem pairwise_on.mono' {s : set α} {r r' : α → α → Prop}
   (H : ∀ a b, r a b → r' a b) (hp : pairwise_on s r) : pairwise_on s r' :=
 λ x xs y ys h, H _ _ (hp x xs y ys h)
 
+/- Disjoint sets and pairwise disjoint collections of sets -/
+lemma disjoint_of_subset {s t s' t' : set α} (hst : s ∩ t = ∅) (hs : s' ⊆ s) (ht : t' ⊆ t) :
```

Comment:
There is already a definition of `disjoint` in `data.set.lattice`, which is preferred over this one. Could you move / merge these lemmas?